examples/kubectl.gpt

```diff
@@ -0,0 +1,42 @@
+tools: sys.write, kubectl, helm, browser
+
+Do not make parallel function calls. Only call one function at a time.
```

Comment:
this and line 5 should no longer be needed as we baked this into the system prompt of gptscript (see: https://github.com/gptscript-ai/gptscript/pull/161/files)

I'll merge this as-is, but I'm curious if you see unexpected behavior without these two lines.